Dnn.AdminExperience/Extensions/Content/Dnn.PersonaBar.Extensions/Services/ServerSettingsSmtpAdminController.cs

```diff
@@ -120,8 +120,8 @@ public HttpResponseMessage SendTestEmail(SendTestEmailRequest request)
             try
             {
                 var smtpHostMode = request.SmtpServerMode == "h";
-                var mailFrom = Host.HostEmail;
-                var mailTo = smtpHostMode ? Host.HostEmail : PortalSettings.UserInfo.Email;
+                var mailFrom = smtpHostMode ? Host.HostEmail : PortalSettings.Email;
+                var mailTo = PortalSettings.Current.UserInfo.Email;
```

Comment:
var mailFrom = smtpHostMode ? Host.HostEmail : PortalSettings.Email;

If I am reading this statement correctly:

If smtHostMode is "=" TRUE then use host email alternatively ":" use use Portal Admin Email

The statement above this sets smtpHostMode will be true when smtpHostMode == h , otherwise it will be false.

var smtpHostMode = request.SmtpServerMode == "h";

"==" should return a true/false boolean datatype value.

so if SmtpServerMode is false it should use the alternate address

A really slick if statement packed in one line.